selfdrive/car/hyundai/carstate.py

```diff
@@ -45,63 +45,72 @@ def get_can_parser(CP):
     ("CF_Clu_AmpInfo", "CLU11", 0),
     ("CF_Clu_AliveCnt1", "CLU11", 0),
 
-    ("CF_Clu_InhibitD", "CLU15", 0),
-    ("CF_Clu_InhibitP", "CLU15", 0),
-    ("CF_Clu_InhibitN", "CLU15", 0),
-    ("CF_Clu_InhibitR", "CLU15", 0),
-
-    ("CF_Lvr_Gear", "LVR12",0),
-    ("CUR_GR", "TCU12",0),
-
     ("ACCEnable", "TCS13", 0),
     ("ACC_REQ", "TCS13", 0),
+    ("BrakeLight", "TCS13", 0),
     ("DriverBraking", "TCS13", 0),
     ("DriverOverride", "TCS13", 0),
 
     ("ESC_Off_Step", "TCS15", 0),
 
     ("CF_Lvr_GearInf", "LVR11", 0),        #Transmission Gear (0 = N or P, 1-8 = Fwd, 14 = Rev)
 
-    ("CR_Mdps_DrvTq", "MDPS11", 0),
-
     ("CR_Mdps_StrColTq", "MDPS12", 0),
     ("CF_Mdps_ToiActive", "MDPS12", 0),
     ("CF_Mdps_ToiUnavail", "MDPS12", 0),
     ("CF_Mdps_FailStat", "MDPS12", 0),
     ("CR_Mdps_OutTq", "MDPS12", 0),
 
+    ("SAS_Angle", "SAS11", 0),
+    ("SAS_Speed", "SAS11", 0),
+
+    ("MainMode_ACC", "SCC11", 0),
     ("VSetDis", "SCC11", 0),
     ("SCCInfoDisplay", "SCC11", 0),
+    ("ACC_ObjDist", "SCC11", 0),
     ("ACCMode", "SCC12", 1),
-
-    ("SAS_Angle", "SAS11", 0),
-    ("SAS_Speed", "SAS11", 0),
   ]
 
   checks = [
     # address, frequency
     ("MDPS12", 50),
-    ("MDPS11", 100),
     ("TCS15", 10),
-    ("TCS13", 50),
```

Comment:
There are still things depending on this signal (TCS13). You can't remove the check

Reply:
this is HKG stuff, some car do not have this signal. I can go back to see what car was 
it is not my car otherwise, I would have tried to find the alternative msg

Reply:
But the TCS13 signal is used above to determine things. See line 166 for example. If you want car state to depend on the signal, you need to check to make sure it's valid, otherwise car state is wrong!

Reply:
hyundai HYUNDAI SANTA FE without SCC does not have this message
I will revert this change and this car will no longer supported

Reply:
We don't support cars without SCC

Reply:
fixed